### PR TITLE
Readme Updated

### PR DESCRIPTION
**Pull request by ayush461**
## Related Issue 

Changed the readme format.

Closes: #60
### Describe the changes you've made

I have changed the readme format as per rules.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,125 +1,208 @@
...
+![](https://img.shields.io/badge/express.js%20-%23404d59.svg?&style=for-the-badge)
+![](https://img.shields.io/badge/MongoDB-%234ea94b.svg?&style=for-the-badge&logo=mongodb&logoColor=white)
+
 We are primarily considering JavaScript (sorry, not TypeScript) as the base language. We are using MERN Stack for this, where the tech stack consists of:
 
-* Bootstrap 5 for the Front End
-* React JS with both Class & Functional Components
-* Node JS & Express JS
-* MySQL or PostgreSQL (yet to be decided)
+Front End
```
Comment (praveenscience) [inline README.md:83]: Shall we make it bold? What do you suggest?

File: README.md [hunk 2]
```diff
@@ -1,125 +1,208 @@
...
+<!--UL -->
+* HTML & CSS(SCSS)
+* Bootstrap 4 & Responsiveness
+* JavaScript & ES6
+* ReactJS & Node JS(lil bit for runtime)
 
-> #### Important Note:
->
-> The typical MERN Stack involves MongoDB + ExpressJS + ReactJS + NodeJS. But, if you consider our product, it really needs a hierarchical database management system and not a document oriented one. MongoDB doesn't perform well under a lot of situations, so we're planning to use MySQL (or MariaDB) or PostgreSQL (We have already mentioned that it's yet to be decided between these two]. I hope this note helps people understand why we are not using MongoDB in MERN Stack.
+Back End
```
Comment (praveenscience) [inline README.md:90]: Shall we make it bold? What do you suggest?

File: README.md [hunk 3]
```diff
@@ -1,125 +1,208 @@
...
 
-### Project Landing Page: Winter of Code
+* Node JS + Express JS
+* Morgan + CORS & Node Mon
+* PostgreSQL/MySQL
 
-I am currently building the landing page for the Winter of Code. It's going on in a steady state and I'll let everyone know as soon as possible once this is done! So far, I have got till here, I hope it looks good:
+Deployment
```
Comment (praveenscience) [inline README.md:96]: Shall we make it bold? What do you suggest?

File: README.md [hunk 4]
```diff
@@ -1,125 +1,208 @@
...
 
-## Interested?
+>#### Important Note:
+>
+>The typical MERN Stack involves MongoDB + ExpressJS + ReactJS + NodeJS. But, if you consider our product, it really needs a hierarchical database management system and not a document oriented one. MongoDB doesn't perform well under a lot of situations, so we're planning to use MySQL (or MariaDB) or PostgreSQL (We have already mentioned that it's yet to be decided between these two]. I hope this note helps people understand why we are not using MongoDB in MERN Stack.
 
-Contributions to this project are currently welcome through one of the following channels:
+## Installation & Running
+## Team
+### Admin & Mentors
```
Comment (praveenscience) [inline README.md:108]: Please add one line space under the first two. 😅

File: README.md [hunk 5]
```diff
@@ -1,125 +1,208 @@
...
+* Gaurav Prajapati
+* Salma Shaik
+* Prachi Mane
+
+### Contributors List
 
-Please read the [Contributing guidelines](https://github.com/vismitap/Internship-LMS-FrontEnd/blob/main/CONTRIBUTING.md) before contributing!
+<a href="https://github.com/praveenscience/Internship-LMS-FrontEnd/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=praveenscience/Internship-LMS-FrontEnd" />
+</a>
```
Comment (praveenscience) [inline README.md:133]: Make it into Markdown.

File: README.md [hunk 6]
```diff
@@ -1,125 +1,208 @@
...
 
 We are in the mission of solving a crisis for a lot of small and medium enterprises, who are trying to recruit interns for their projects. This platform aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.
 
-> **Winter of Code Series!**
-> 
-> Check out the [Winter of Code](https://www.youtube.com/playlist?list=PLO_Y0rsm7b3Yc8FtNhCuD5TO0QM8CnTZj) Series on YouTube. It will give a great start for you, if you're a complete fresher! Also, I would kindly request you to subscribe to my [YouTube Channel](https://www.youtube.com/praveenscience?sub_confirmation=1), so that you'll be getting notifications of awesome things!
+### Motivation
 
-To start contributing, directly jump to the [**Steps to Contribute**](#steps) and follow everything. In case, if you're unsure, please check out the [FAQs](#faqs) (Frequently Asked Questions).
+- Make the process smooth.
```
Comment (praveenscience) [inline README.md:18]: Where's the automation line? 🤔

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -1,125 +1,208 @@
...
+## Installation & Running
+## Team
+### Admin & Mentors
 
-* [DevScript Winter of Code](https://devscript.tech/woc/)
-* [Script Winter of Code](https://swoc.tech/)
+* Praveen Kumar Purushothaman ([Website](https://praveen.science/) &#1421; [LinkedIn](https://www.linkedin.com/in/praveentech/) &#1421; [YouTube](https://www.youtube.com/praveenscience?sub_confirmation=1))
+* Shashwat Mishra ([LinkedIn](https://www.linkedin.com/in/the-shashwat-mishra/))
+* Aditya Pahilwani ([LinkedIn](https://www.linkedin.com/in/aditya-pahilwani-93b42bb5/))
+* Tejan Singh([LinkedIn](https://www.linkedin.com/in/tejansingh/))
```
[the remark this answers is not in the document]
Comment (ayush461) [reply README.md:113]: @praveenscience sir, spaces should be after every line or only after first line?

[the remark this answers is not in the document]
Comment (praveenscience) [reply README.md:115]: @ayush461 After `Singh` 😅

**Review by praveenscience — APPROVED** [review]
I'll make it right. 😊

**Review by praveenscience — APPROVED** [review]
LGTM. I fixed all the issues man! 😊

**Pull request merged**